.github/workflows/deploy_bookdown.yml

```
@@ -0,0 +1,52 @@
+on:
+  push:
+     branches:
+       - master
+
+  
+
+name: renderbook
+
+jobs:
+  bookdown:
+    name: Render-Book
+    runs-on: macOS-latest
```

```suggestion
    runs-on: ubuntu-latest
```
in general, Ubuntu runners are faster and cheaper.